tests/integration/output/local_log/run_MotEvo_analysis.motif_HNRNPF_822.benchmark.log

```diff
@@ -0,0 +1,2 @@
+s	h:m:s	max_rss	max_vms	max_uss	max_pss	io_in	io_out	mean_load
```

Comment:
The same situation here - this is a log file.
This applies to all the `stderr` and `stdout` files as well(!)